src/verify.rs

```diff
@@ -197,6 +202,64 @@ pub fn verify_server_cert(roots: &RootCertStore,
     .map_err(|err| TLSError::WebPKIError(err))
 }
 
+pub fn parallel_verify_server_cert(roots: &RootCertStore,
+                                   presented_certs: &Vec<ASN1Cert>,
+                                   dns_name: &str) -> Result<(), TLSError> {
+  crossbeam::scope(|scope| {
```

Comment:
How would the Rustls user control the level of parallelism (how many threads are used)?